lib/srv/sess.go

```diff
@@ -363,10 +506,13 @@ func newSession(id rsession.ID, r *SessionRegistry, ctx *ServerContext) (*sessio
 		// return nil, trace.Wrap(err)
 		// No need to abort. Perhaps the auth server is down?
 		// Log the error and continue:
-		log.Errorf("failed logging new session: %v", err)
+		r.log.Errorf("Failed to create new session: %v.", err)
 	}
 
 	sess := &session{
+		log: logrus.WithFields(logrus.Fields{
+			trace.Component: fmt.Sprintf("%v:%v", teleport.ComponentSession, r.srv.Component()),
```

Comment:
there is a function for that: `teleport.Component` that constructs components vs using sprintf directly